src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -66,7 +70,7 @@ def do_strategy(strategy_tuple):
                                              strategy_tuple.probability))
 
 
-def generate_strategy_pool():
+def generate_random_strategy_pool():
```

Comment:
nit: not sure if "random" is a good adjective here, as both pool versions still "random", just use different probabilities. How about we call this "default" and the other one "weighted"?

Reply:
good idea